compareECLFiles.cmake

```diff
@@ -37,9 +37,38 @@ function(add_test_runSimulator)
                            -f ${PARAM_FILENAME}
                            -n ${PARAM_PROCS}
                TEST_ARGS ${TEST_ARGS}
-               CONFIGURATION extra)
+               CONFIGURATION ${PARAM_CONFIGURATION})
 endfunction()
 
+###########################################################################
```

Comment:
this macro seems to be unused, and not really necessary. looks like remnants from earlier attempts.

Reply:
Ah, you're right 😳  - deleting.